.changes/next-release/Bug Fix-291dc161-1ef0-4a28-9256-df2d9a5b5542.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fixed bug when CodeWhisperer left/right arrow keybinding conflict with other plugins"
```

The type is "Bug Fix", so all that's needed is a description of the bug fix.

```suggestion
	"description": "CodeWhisperer left/right arrow keybinding conflict with other plugins"
```